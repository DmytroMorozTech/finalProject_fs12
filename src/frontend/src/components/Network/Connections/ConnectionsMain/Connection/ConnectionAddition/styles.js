import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  menuItem: {
    '&:hover': {
      backgroundColor: 'rgba(0, 0, 0, 0.08)'
    }
  },

  item: {
    display: 'flex',
    flexDirection: 'row',
    alignItems: 'center',
    fontSize: theme.typography.h5.fontSize,
    fontWeight: theme.typography.fontWeightBold,
    color: theme.palette.grey[700],
    cursor: 'pointer'
  },

  icon: {
    display: 'flex'
  },

  titles: {
    paddingLeft: theme.spacing(2)
  }
}))
