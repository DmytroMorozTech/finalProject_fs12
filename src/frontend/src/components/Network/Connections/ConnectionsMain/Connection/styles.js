import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  connection: {
    display: 'flex',
    marginLeft: theme.spacing(5),
    marginRight: theme.spacing(5),
    paddingTop: theme.spacing(2),
    paddingBottom: theme.spacing(2),
    justifyContent: 'space-between'
  },

  removed: {
    display: 'none'
  },

  flex: {
    display: 'flex'
  },

  avatar: {
    width: theme.avatar.large,
    borderRadius: '50%'
  },

  userInfo: {
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'center',
    marginLeft: theme.spacing(2)
  },

  link: {
    textDecoration: 'none',
    fontSize: theme.typography.sh3.fontSize,
    fontWeight: theme.typography.fontWeightBold,
    color: theme.palette.grey[700],
    '&:hover': {
      textDecoration: 'underline',
      textDecorationThickness: '2px',
      textDecorationColor: theme.palette.grey[700]
    }
  },

  buttons: {
    display: 'flex',
    alignItems: 'center'
  },

  button: {
    marginRight: theme.spacing(2)
  },

  line: {
    width: 'calc(100% - 100px)',
    height: '0.5px',
    border: '0',
    margin: '0',
    marginLeft: '100px',
    backgroundColor: theme.palette.grey[300]
  },

  linkButton: {
    textDecoration: 'none'
  }
}))
