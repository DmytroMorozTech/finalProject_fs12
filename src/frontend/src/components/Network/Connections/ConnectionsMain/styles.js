import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  connectionsMain: {
    border: theme.border.simple,
    borderRadius: theme.shape.medium,
    backgroundColor: theme.palette.common.white
  },

  title: {
    padding: theme.spacing(2),
    paddingLeft: theme.spacing(4),
    color: theme.palette.grey[700],
    fontWeight: theme.typography.fontWeightRegular
  },

  line: {
    width: '100%',
    height: '0.5px',
    border: '0',
    margin: '0 auto',
    backgroundColor: theme.palette.grey[300]
  }
}))
