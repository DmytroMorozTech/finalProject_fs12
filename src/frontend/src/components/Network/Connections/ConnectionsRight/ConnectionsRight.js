import styles from './styles'
import ImgToJobsPage from '../../../../shared/ImgToJobsPage/ImgToJobsPage'
import React from 'react'

function ConnectionsRight () {
  const classes = styles()

  return (
    <div className={classes.connectionsRight}>
      <ImgToJobsPage/>
    </div>
  )
}

export default ConnectionsRight
