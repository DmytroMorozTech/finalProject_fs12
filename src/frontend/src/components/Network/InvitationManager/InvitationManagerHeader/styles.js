import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  invitationManagerHeader: {
    paddingTop: theme.spacing(2),
    paddingLeft: theme.spacing(4)
  },

  title: {
    fontWeight: theme.typography.fontWeightRegular
  },

  headerItems: {
    display: 'flex',
    marginTop: theme.spacing(2)
  },

  headerItem: {
    marginTop: theme.spacing(3),
    marginRight: theme.spacing(6),
    paddingBottom: theme.spacing(1),
    cursor: 'pointer',
    textDecoration: 'none',
    color: theme.palette.grey[700]
  },

  active: {
    color: theme.palette.success.main,
    borderBottom: theme.border.success
  }
}))
