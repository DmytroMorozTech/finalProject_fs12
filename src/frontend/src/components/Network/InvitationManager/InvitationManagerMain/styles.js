import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  blockNoInvitations: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    paddingTop: theme.spacing(3),
    paddingBottom: theme.spacing(8),
    borderTop: theme.border.simple
  },

  image: {
    height: '128px',
    width: '128px'
  },

  defaultText: {
    fontWeight: theme.typography.fontWeightRegular,
    paddingTop: theme.spacing(4)
  }
}))
