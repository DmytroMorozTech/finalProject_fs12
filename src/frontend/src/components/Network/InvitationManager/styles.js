import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  invitationManager: {
    marginTop: '75px'
  },

  main: {
    border: theme.border.simple,
    borderRadius: theme.shape.medium,
    backgroundColor: theme.palette.common.white,
    overflow: 'hidden'
  }
}))
