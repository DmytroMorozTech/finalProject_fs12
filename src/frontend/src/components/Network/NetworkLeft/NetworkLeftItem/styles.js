import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  container: {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'space-between',
    padding: theme.spacing(2),
    paddingLeft: theme.spacing(4),
    paddingRight: theme.spacing(4),
    cursor: 'pointer',
    textDecoration: 'none',
    '&:hover': {
      backgroundColor: 'rgba(0,0,0,0.1)'
    }
  },

  items: {
    display: 'flex',
    flexDirection: 'row',
    alignItems: 'center',
    fontSize: theme.typography.h5.fontSize,
    cursor: 'pointer'
  },

  icons: {
    display: 'flex',
    fontSize: theme.typography.icons.large.fontSize,
    color: theme.palette.grey[700]
  },

  titles: {
    paddingLeft: theme.spacing(2),
    color: theme.palette.grey[600]
  },

  number: {
    color: theme.palette.grey[600]
  }
}))
