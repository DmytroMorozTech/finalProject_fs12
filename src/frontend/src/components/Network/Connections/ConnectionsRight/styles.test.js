import React from 'react'
import ConnectionsRight from './ConnectionsRight'
import { render } from 'enzyme'
import theme from '../../../../Theme/Theme'
import { ThemeProvider } from '@material-ui/styles'

jest.mock('react-router-dom', () => ({
  Link: 'a'
}))

const setUp = () => render(
  <ThemeProvider theme={theme}>
    <ConnectionsRight />
  </ThemeProvider>)

describe('should render ConnectionsRight styles', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
