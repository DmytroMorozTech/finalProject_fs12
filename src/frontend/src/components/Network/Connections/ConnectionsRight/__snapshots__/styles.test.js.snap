// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ConnectionsRight styles to match snapshot 1`] = `undefined`;
